Dockerfile

```diff
@@ -52,21 +56,23 @@ WORKDIR /
 ########################
 FROM psyllid_common as psyllid_done
 
-COPY dripline-cpp /tmp_source/dripline-cpp
 COPY external /tmp_source/external
-COPY midge /tmp_source/midge
 COPY monarch /tmp_source/monarch
+COPY sandfly /tmp_source/sandfly
 COPY source /tmp_source/source
 COPY CMakeLists.txt /tmp_source/CMakeLists.txt
 COPY .git /tmp_source/.git
+COPY .gitmodules /tmp_source/.gitmodules
+COPY PsyllidConfig.cmake.in /tmp_source/PsyllidConfig.cmake.in
 
 ## store cmake args because we'll need to run twice (known package_builder issue)
 ## use EXTRA_CMAKE_ARGS to add or replace options at build time, CMAKE_CONFIG_ARGS_LIST are defaults
 ARG EXTRA_CMAKE_ARGS=""
 ENV CMAKE_CONFIG_ARGS_LIST="\
-      -D CMAKE_INSTALL_PREFIX:PATH=$PSYLLID_BUILD_PREFIX \
+      -D CMAKE_INSTALL_PREFIX:PATH=$SANDFLY_BUILD_PREFIX \
```

Comment:
I think that change is not right. That variable was never set anywhere? And consequently after building the container, psyllid is not installed where it should be.

Reply:
I pushed a fix for that with fbbddb6